code/game/machinery/droneDispenser.dm

```diff
@@ -125,6 +125,8 @@
 
 /obj/machinery/drone_dispenser/examine(mob/user)
 	. = ..()
+	if (iron_cost > 0 || glass_cost > 0)
+		. += span_notice("It needs [iron_cost] iron and [glass_cost] glass to produce one drone shell.")
```

Comment:
You check if it needs either iron or glass, but then the text implies it will require both?

Reply:
I'm not actually sure how to code it in a way that makes only 1 material appears. As far as I'm aware . += on examine will always make a new line. I don't want it to say "It needs 1000 iron to produce one drone shell" (newline) "It needs 1000 glass to produce one drone shell"

Reply:
Nevermind I though or a way